Countries/.common/legal-privacy-policy.html

```diff
@@ -255,3 +255,4 @@ <h2>Changes to This Privacy Policy</h2>
 
   <h2>Contact Us</h2>
   <p>If you have any questions about this Privacy Policy, please contact us.</p>
+</p>
```

Comment:
I think this `</p>` should be on line 74, having `h`, `ul` and `p` elements inside `p` isn't valid html.